frontend/src/views/VotePage.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <form v-if="election !== null" @submit.prevent="voteInElection()">
+    <h1>{{ election.title }}</h1>
+    <p>{{ election.description }}</p>
+    <div v-for="(question, i) in election.questions" :key="i">
+      <p>Question #{{ i }}</p>
+      <div v-for="(answer, ai) in question.answers" :key="ai">
+        <input
+          required
+          type="radio"
+          :id="i + '.' + ai"
```

Comment:
Would `` `${i}.${ai}` `` work for it? I like these template literals in js

Reply:
Nice, I didn't think of using it here. If it works, I'll add it in the next PR 👍